### PR TITLE
Ready: Rewrite to clarify and add jQuery 3.0 notes

### LINKED ISSUE #190: new signature in 1.9 for $.fn.css: accepts an array to retrieve a map of property names to values
from this: http://bugs.jquery.com/ticket/11938
as implemented: https://github.com/jquery/jquery/commit/ef6d8779de9d5959eafc0b9706b90a585aaf7a26

Please see the tests for the expected behavior.

### LINKED ISSUE #964: .load() method recommended on .ready() page, even though it's deprecated
In the yellow warning box on http://api.jquery.com/ready/, the `.load()` method is recommended, even though it has been removed in 3.0.0, and was deprecated in 1.8.

### PR DESCRIPTION
**Pull request by dmethvin**
Fixes #964 (EDIT: corrected ticket)

Sorry but I still don't have a working docs setup. I hope I got all the markup right, if not let me know. 

I'm hoping to have a Mac within a few weeks and maybe I can get things working on that.

### CODE REVIEW

File: entries/ready.xml [hunk 1]
```diff
@@ -9,49 +9,53 @@
...
+    <p>The <code>.ready()</code> method offers a way to run JavaScript code as soon as the page's Document Object Model (DOM) becomes safe to manipulate. This will often be a good time to perform tasks that are needed before the user views or interacts with the page, for example to add event handlers and initialize plugins. When multiple functions are added via successive calls to this method, they run when the DOM is ready in the order in which they are added. As of jQuery 3.0, jQuery ensures that an exception occuring in one handler does not prevent subsequently added handlers from executing.</p>
+
+    <p>Most browsers <a href="http://caniuse.com/#search=DOMContentLoaded">provide similar functionality</a> in the form of a <code>DOMContentLoaded</code> event. However, jQuery's <code>.ready()</code> method differs in an important and useful way: If the DOM becomes ready and the browser fires <code>DOMContentLoaded</code> before the code calls <code>.ready( handler )</code>, the function <code>handler</code> will still be executed. In contrast, a <code>DOMContentLoaded</code> event listener added after the event fires is never executed.<p>
```
Comment (kswedberg) [inline entries/ready.xml]: You have `<p>` for the closing paragraph tag. Should be `</p>`.

File: entries/ready.xml [hunk 2]
```diff
@@ -9,49 +9,53 @@
...
+
+    <p>Browsers also provide the <code>load</code> event on the <code>window</code> object. When this event fires it indicates that all assets on the page have loaded, including images. This event can be watched in jQuery using <code>$( window ).on( "load", handler )</code>. In cases where code relies on loaded assets (for example, if the dimensions of an image are required), the code should be placed in a handler for the <code>load</code> event instead.
+
```
Comment (kswedberg) [inline entries/ready.xml]: Needs closing `</p>`

File: entries/ready.xml [hunk 3]
```diff
@@ -9,49 +9,53 @@
...
+    <p>Browsers also provide the <code>load</code> event on the <code>window</code> object. When this event fires it indicates that all assets on the page have loaded, including images. This event can be watched in jQuery using <code>$( window ).on( "load", handler )</code>. In cases where code relies on loaded assets (for example, if the dimensions of an image are required), the code should be placed in a handler for the <code>load</code> event instead.
+    
+    Note that although the DOM always becomes ready before the page is fully loaded, it is <em>usually not safe</em> to attach a <code>load</code> event listener in code executed during a <code>.ready()</code> handler. For example, scripts can be loaded dynamically long after the page has loaded using methods such as <code>$.getScript()</code>. Although handlers added by <code>.ready()</code> will always be executed in a dynamically loaded script, the <code>window</code>'s <code>load</code> event has already occurred and those listeners will never run.
```
Comment (kswedberg) [inline entries/ready.xml]: Wrap this in `<p> … <p>`

**Comment by dmethvin** [comment]
Thanks Karl! :cake:

**Pull request closed without merging**